bigtable-client-core-parent/bigtable-client-core/pom.xml

```diff
@@ -224,7 +224,7 @@ this is not used directly, but upgrading due to transitive vulnerabilities in ol
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
       <!-- align version with veneer's flattened dep -->
-      <version>2.11.1</version>
+      <version>2.12.1</version>
```

Comment:
I think this will break as per comment above

Reply:
veneer uses 2.12.1 now....this is just catching up to it